dbt/adapters/trino/impl.py

```diff
@@ -39,3 +39,6 @@ def convert_datetime_type(cls, agate_table, col_idx):
     @classmethod
     def convert_date_type(cls, agate_table: agate.Table, col_idx: int) -> str:
         return "DATE"
+
+    def timestamp_add_sql(self, add_to: str, number: int = 1, interval: str = "hour") -> str:
```

Comment:
Could you please point out how it's used.

Reply:
This is used here: https://github.com/dbt-labs/dbt-core/blob/1df713fee986741289568f553befdd4e38a51122/core/dbt/tests/util.py#L423 as part of the snapshot integration tests.